scripts/submit-analytics.js

```diff
@@ -52,7 +52,7 @@ let doLog; // placeholder for logger function
  * Path and credentials required to submit analytics through the webhook.
  */
 const ANALYTICS_BASE_URL =
-  "https://webhooks.mongodb-realm.com/api/client/v2.0/app/realmsdkmetrics-zmhtm/service/metric_webhook/incoming_webhook/metric";
+  "https://webhooks.mongodb-realm.com/api/client/v2.0/app/realmsdkmetrics-zmhtm/service/metric_webhook/incoming_webhook/metric?data=";
```

Comment:
I'd probably move this URL parameter to the `getAnalyticsRequestUrl`.

```suggestion
  "https://webhooks.mongodb-realm.com/api/client/v2.0/app/realmsdkmetrics-zmhtm/service/metric_webhook/incoming_webhook/metric";
```